gensrc/thrift/PlanNodes.thrift

```diff
@@ -222,6 +222,18 @@ struct TEsScanNode {
     // {"city": "city.raw"}
     // use select city from table, if enable the docvalue, we will fetch the `city` field value from `city.raw`
     3: optional map<string, string> docvalue_context
+    // used to indicate which string-type field predicate should used xxx.keyword etc.
+    // "k1": {
+    //    "type": "text",
+    //    "fields": {
+    //        "keyword": {
+    //            "type": "keyword",
+    //            "ignore_above": 256
+    //           }
+    //    }
+    // }
+    // k1 > 'abc' -> k1.keyword > 'abc'
+    4: optional map<string, string> fields_context
```

Comment:
If this is only used for keyword path, `fields_context` seems no relation with keyword

Reply:
https://www.elastic.co/guide/en/elasticsearch/reference/current/multi-fields.html

fields is what it means to be, maybe enable_keyword_sniff is no t suitable?

Reply:
You're right